packages/styles/panel.css

```diff
@@ -3,6 +3,22 @@
   --panel-background-color: var(--white);
   --panel-heading-color: var(--gray-90);
   --panel-padding: var(--space-small);
+  --panel-content-padding: var(--space-small);
+  --panel-content-margin: 0 calc(var(--space-small) * -1);
+  --panel-header-margin: calc(var(--space-smaller) * -1)
```

Comment:
These should inherit from appropriate variable value, otherwise they won't match up if the padding changes.